app/src/main/res/layout/fragment.xml

```diff
@@ -0,0 +1,181 @@
+            app:layout_constraintTop_toTopOf="parent"
+            app:srcCompat="@drawable/movie_image" />
+
+        <View
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:background="@drawable/image_mask"
+            app:layout_constraintBottom_toBottomOf="@+id/iv_movies_list_image"
+            app:layout_constraintEnd_toEndOf="@+id/iv_movies_list_image"
+            app:layout_constraintStart_toStartOf="@+id/iv_movies_list_image"
+            app:layout_constraintTop_toTopOf="@+id/iv_movies_list_image" />
+
+        <TextView
+            android:id="@+id/tv_age_limit"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginTop="8dp"
+            android:background="@drawable/ic_rectangle"
+            android:gravity="center"
+            android:text="@string/age"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="@id/left_fragment_guideline"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageView
+            android:id="@+id/iv_like"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:src="@drawable/like"
+            app:layout_constraintBottom_toBottomOf="@id/tv_age_limit"
+            app:layout_constraintEnd_toStartOf="@id/right_fragment_guideline"
+            app:layout_constraintTop_toTopOf="@id/tv_age_limit" />
+
+        <TextView
+            android:id="@+id/tv_movies_list_genre"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
+            android:text="@string/genre"
+            android:textColor="@color/red"
+            android:textSize="8sp"
+            app:layout_constraintBottom_toTopOf="@+id/tv_movie_list_review"
+            app:layout_constraintEnd_toStartOf="@id/right_fragment_guideline"
+            app:layout_constraintStart_toStartOf="@id/left_fragment_guideline" />
+
+        <ImageView
+            android:id="@+id/ic_rating_star_1"
+            android:layout_width="8dp"
+            android:layout_height="0dp"
+            android:src="@drawable/star_icon"
+            app:layout_constraintStart_toStartOf="@id/left_fragment_guideline"
+            app:layout_constraintTop_toTopOf="@+id/tv_movie_list_review"
+            app:tint="@color/red"
+            tools:ignore="ContentDescription" />
+
+        <ImageView
+            android:id="@+id/ic_rating_star_2"
+            android:layout_width="8dp"
+            android:layout_height="0dp"
+            android:layout_marginStart="2dp"
+            android:src="@drawable/star_icon"
+            app:layout_constraintStart_toEndOf="@+id/ic_rating_star_1"
+            app:layout_constraintTop_toTopOf="@+id/ic_rating_star_1"
```

Comment:
app:layout_constraintBottom_toBottomOf="@+id/ic_rating_star_1"

for 2,3,4,5 stars